flocker/ca/_ca.py

```diff
@@ -332,6 +355,36 @@ class UserCredential(PRecord):
     username = field(mandatory=True, type=unicode)
 
     @classmethod
+    def from_files(cls, certificate_path, key_path):
```

Comment:
At least change the method names / docstrings to present this information more clearly.